README.md

```diff
@@ -1,16 +1,17 @@
 # Easy-YT
-Making the process of downloading youtube videos faster and more convinient.
-
-# What can it do?
-This python script can be used to download youtube vidoes in either mp4 or mp3 format.
-Any number of videos can be downloaded at a single time
-
-# Is the project Completed?
-Pretty much yes. I felt lazy to find bugs but it works perfectly
-
-# Requirements
-selenium (pip install selenium)<br />
-pytube (pip install pytube)<br />
-chromedrvier (https://chromedriver.chromium.org/downloads) (Download correct driver with version similar to your chrome browser)
+This Python program simplifies the process of downloading videos off of [http://youtube.com/](YouTube) while also increasing download speed.
+The script can be used to download vidoes in either MP4(Video) or MP3(Audio) formats. Multiple videos can be downloaded at the same time.
+| The program works perfectly but there may be edge cases I have yet to pin down.
 
+# Getting Started
+## Requirements
+1. selenium `pip install selenium`
+2. pytube `pip install pytube`
+3. chrome driver [https://chromedriver.chromium.org/downloads](Download correct driver) with version similar to your chrome browser.
 
+That's it! Now you can run the script:
+```
+git clone https://github.com/raheelrjunaid/Easy-YT.git
```

Comment:
I meant to git clone https://github.com/meyman/Easy-YT.git

Reply:
Thanks :)